packages/playwright/src/reporter.ts

```diff
@@ -78,9 +78,35 @@ export interface ReplayPlaywrightConfig
 
 class ReplayPlaywrightReporter implements Reporter {
   reporter?: ReplayReporter<ReplayPlaywrightRecordingMetadata>;
-  captureTestFile = ["1", "true"].includes(
-    process.env.PLAYWRIGHT_REPLAY_CAPTURE_TEST_FILE?.toLowerCase() || "true"
-  );
+  captureTestFile: boolean;
+  config: ReplayPlaywrightConfig;
+
+  constructor(config: ReplayPlaywrightConfig) {
```

Comment:
Playwright introduced Playwright Test (and thus reporters) in 1.12.0 and those options were passed to constructors there right from the beginning:
https://github.com/microsoft/playwright/blob/3b669e11eb6ccc05f8987cae71213577394e760e/src/test/runner.ts#L68-L69

I'm not sure why we were getting this in `onBegin` using `parseConfig`.